### PR TITLE
Refactor and Minor Changes

### PR DESCRIPTION
**Pull request by jesusgn90**
Hi Angry Koala!

- All refactored to more ES6 style.
- Changed some if/else to cond ? .. : ..;
- Added typeof .. === function for done callback

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ yerbamate.run("cat my_file.md", function(code, out, errs){
 _Yerbamate_ also provides easy access to you package.json defined scripts and commands, so you can test your module easily with `yerbamate.loadPackage`.
 
 ```js
-var yerbamate=require('yerbamate');
+const yerbamate = require('yerbamate');
 
-var pkg=yerbamate.loadPackage(module);
+let pkg = yerbamate.loadPackage(module);
```
Comment (angrykoala) [inline README.md]: Is there a reason for this being a `let` instead of a `const`?

**Comment by angrykoala** [comment]
Please, do all pull requests against `dev` branch, you can check the contributing guidelines in the [CONTRIBUTING.md](https://github.com/angrykoala/yerbamate/blob/master/CONTRIBUTING.md) file

File: app/loader.js [hunk 2]
```diff
@@ -1,58 +1,67 @@
...
-    return getContents(JSON.parse(contents), path.dirname(filepath));
 }
 
-module.exports = function(pmodule) {
-    if (!pmodule) throw new Error("yerbamate loader - Not module found");
-    if (typeof pmodule === "string") return fileLoader(pmodule);
-    else return moduleLoader(pmodule);
-};
+
+module.exports = Loader.main;
```
Comment (angrykoala) [inline app/loader.js]: Instead of adding a main method in the Loader, export the function directly

**Comment by angrykoala** [comment]
PR cannot be merged due to some conflicts, resolve those conflicts with the `dev` branch please

**Comment by jesusgn90** [comment]
Ok, i created new PR to dev

**Pull request closed without merging**